File: mobileapp/CommonLib/Activity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Elixir.Common
{
    public struct Activity
    {
        public int SessionId { get; set; }
        public DateTime PourTime { get; set; }
        public int PourAmount { get; set; }
        public string BeerName { get; set; }
        public string Brewery { get; set; }
        public string BeerType { get; set; }
        public int ABV { get; set; }
        public int IBU { get; set; }
        public string BeerDescription { get; set; }
        public object UntappdId { get; set; }
        public string BeerImagePath { get; set; }
        public int PersonnelNumber { get; set; }
        public string Alias { get; set; }
        public string FullName { get; set; }
    }
}

File: app/Model/Activity.ts
//     Copyright (c) Microsoft Corporation.  All rights reserved.

/// <reference path="../references/index.d.ts" />

module DXLiquidIntel.App.Model {

    // Need to keep structure in sync with DashServer.ManagementAPI.Models.OperationState in the WebAPI
    export class Activity {
        public SessionId: number
        public PourTime: Date
        public PourAmount: number
        public BeerName: string
        public Brewery: string
        public BeerType: string
        public ABV?: number
        public IBU?: number
        public BeerDescription: string
        public UntappdId?: number
        public BeerImagePath: string
        public PersonnelNumber: number
        public Alias: string
        public FullName: string
    }
}
         